knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -156,6 +156,12 @@ func add(mgr manager.Manager, r *ReconcileKnativeKafka) error {
 		}
 	}
 
+	// watch KnativeEventing instances as KnativeKafka instances are dependent on them
+	err = c.Watch(&source.Kind{Type: &operatorv1beta1.KnativeEventing{}}, &handler.EnqueueRequestForObject{})
```

Comment:
Does EnqueueRequestForObject enqueues KnativeKafka objects? or would it enqueue KnativeEventing?

Reply:
@pierDipi I pushed a new fix now